### DISCUSSION TITLE
Is it possible to edit values for variables defined in config.py via the browser?

### DISCUSSION THREAD

**Discussion by ghjardim** [qutebrowser/qutebrowser#6538] [Q&A] [answered]
Hi! Currently I have in my config.py a code that abstractly does this:
```
redirecting_enabled = True
if redirecting_enabled:
    redirect reddit.com to old.reddit.com
```

Is it possible to change the value for ```redirecting_enabled``` directly via the browser? Is there something different I can do to achieve the same effects without having to restart the browser?

**Comment by The-Compiler** [comment, marked as answer]
No, but you can use `:config-source` to reload your `config.py` - so you probably could have a [userscript](https://github.com/qutebrowser/qutebrowser/blob/master/doc/userscripts.asciidoc) changing the file (or a file [included from there](https://github.com/qutebrowser/qutebrowser/blob/master/doc/help/configuring.asciidoc#importing-other-modules)) and then running that.

That all being said, if your redirects work via registering an interceptor, it's not like reloading the config will make that magically disappear. Doing that isn't supported, so you're on your own there.
